MANUAL.md

```diff
@@ -0,0 +1,111 @@
+# Making a tilegram
+
+A “tilegram” is a map made of tiles
+where regions are sized proportionally to a dataset.
+The name is short for a tiled
+[cartogram](https://en.wikipedia.org/wiki/Cartogram).
+Tilegrams can represent demographic data more truthfully than conventional,
+geographic maps, while still retaining a familiar appearance.
+
+This free, open-source tool enables news designers and developers
+to browse existing tilegrams or make their own for use in interactive
+and print publication.
+
+Even with computer automation, tilegrams can be time-consuming to produce,
+because, to be effective, they require a human eye to verify that geographic
+contours will be recognizable and meaningful to a general readership. For this
+reason, you're encouraged to begin with existing tilegrams before authoring
+your own.
+
+This manual proceeds from the most basic to the most advanced usage case.
+
+## Exporting existing tilegrams
+
+On load, you will see a ready-made tilegram, as selected in the
+**Load existing** menu. Try selecting other options to browse around.
+
+If you are satisfied with the tilegram as it appears, go right ahead and
+**Export**—as **TopoJSON** or **SVG**—using the buttons at the lower left.
+
+Designers will be able to import the SVG into their software of choice
+(e.g. Illustrator) and developers will be able to pull the TopoJSON into
+web applications. In both cases, the data is identified with the US
+[FIPS](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards)
+code.
+
+## Editing tilegrams
+
+Suppose you've loaded a tilegram, but want to reshape a region. Maybe Florida
+looks too heavy, or Missouri is streched too thin.
+
+Click step **2**: **Refine your tilegram**.
+
+### Moving tiles around
+
+Click and drag any tile to move it around.
+
+To move many tiles around, click and drag a
+rectangular marquee around them, and then drag them around.
+
+To move just a specific region around, double-click
+any tile in it to select them all—then drag them around.
+
+You can also hover over a region in the **State Tiles**
+sidebar area to see that region's tiles highlighted on the map.
+
+### Ensuring statistical accuracy
+
+Under **State Tiles**, you'll see a list of each state with a number and a
+hexagon.
+
+The number indicates the _delta inaccuracy_ between the number of tiles that
+region _currently_ has on the map and how many it _should_ have, based on the
+dataset. If the delta is positive, that region has too many tiles on the map.
+If the delta is negative, it doesn't have enough tiles on the map. If there
+is a warning sign, then that region has _no_ tiles on the map; that may be
```

Comment:
"If there is a warning sign, then that region doesn't have enough data to be statistically represented on the map at the chosen resolution." Too complicated? Technically the warning shows for geo regions that don't round to at least one tile.